### DISCUSSION TITLE
Support for Python3.13

### DISCUSSION THREAD

**Discussion by ishanasse** [scrapli/scrapli_netconf#145] [Q&A]
Python 3.13 has been released.
The latest version of "lxml 5.3.0" can be installed on Python3.13, but "scrapli_netconf" has locked this dependency  lxml<5.0.0, due to which it is not possible to install it on Python3.13

```
Collecting scrapli_netconf
  Using cached scrapli_netconf-2024.7.30-py3-none-any.whl.metadata (8.3 kB)
Collecting lxml<5.0.0,>=4.5.1 (from scrapli_netconf)
```

Thank you in advance.

**Comment by carlmontanari** [comment]
pr is welcome if you wanna put one together
